commands/utils/batch/batch_runner.js

```diff
@@ -113,7 +115,22 @@ async function run(lt_config, batches, env, i = 0) {
                       .then(function (session_id) {
                         delete_archive(project_file);
                         delete_archive(file_obj["name"]);
-
+                        //listen to control+c signal and stop tests
+                        process.on("SIGINT", async () => {
+                          try {
+                            console.log(
+                              "Control+c signal recieved.\nTrying to Terminating the processes"
```

Comment:
@japneetlambdatest `received. Trying to terminate the processes`. Use of single present continuous verb in single sentence.